modules/data-storage-management.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/optimizing-storage.adoc
+
+[id="data-storage-management_{context}"]
+= Data storage management
+
+The following table summarizes the main directories that {product-title} components write data to.
+
+.Main directories for storing {product-title} data
+[options="header,footer"]
+|===
+|Directory|Notes|Sizing|Expected growth
+
+|*_/var/lib/openshift_*
+|Used for etcd storage only when in single master mode and etcd is embedded in the *atomic-openshift-master* process.
+|Less than 10 GB.
+|Will grow slowly with the environment. Only storing metadata.
```

Comment:
I think that we do not use this directory any longer.

Reply:
etcd in 4.x uses /var/lib/etcd for storing the database, which can grow up to 8GB.

Reply:
@retroflexer - Thanks, I modified `/var/lib/etcd` row based on your comment (see changes wrapped in `**` below:
```
|*_/var/lib/etcd_*
|Used for etcd storage when in Multi-Master mode, when etcd is made standalone by an administrator, **or when storing the database**.
|Less than 20 GB.

**Database can grow up to 8 GB.**
|Will grow slowly with the environment. Only storing metadata.

Additional 20-25 GB for every additional 8 GB of memory.
```